web/modules/custom/vote_utility/translations/nv.po

```diff
@@ -13,214 +8,285 @@ msgstr ""
 msgstr "N1t'33'"
 
-msgid "Illustration of voter box with checked ballot being inserted to indicate voting action"
-msgstr "&'ii'n77[ b1 sits'aa y1zh7 si'1n7g77 biyi' asdzoh naashch'22'7g77 47 n7l47 o'oot'ah n7igo yaa y1[ti'go si'32go 1lyaa"
-
+# config.languages.params.last_updated
 msgid "Last updated:"
-msgstr "Ak44'di n1zk'33z:"
+msgstr "Ak44'di n1zk'33z: "
+
+# config.languages.params.ext_link_title
+msgid "External link opens in new window"
+msgstr "External link opens new window"
 
-#identifier
+# config.languages.params.skip_text
+msgid "Skip to main content"
+msgstr "&yis77 baa hane'7gi nin7[keed"
+
+# config.languages.params.english_only
+msgid "(in English)"
+msgstr "&lt;Bilag1ana k'ehj7&gt;"
+
+# data.translations.footer.vote_logo_alt_text
+msgid "Illustration of voter box with checked ballot being inserted to indicate voting action"
+msgstr "&'ii'n77[ b1 sits'aa y1zh7 si'1n7g77 biyi' asdzoh naashch'22'7g77 47 n7l47 o'oot'ah n7igo yaa y1[ti'go si'32go 1lyaa "
+
+# data.translations.footer.identifier_gsa_txt
 msgid "An official website of the @link."
 msgstr "!l32j8' han1lki'7g77 website kod00 @link"
 
+# data.translations.footer.identifier_gsa_txt__1
 msgid "General Services Administration"
 msgstr "General Services Administration woly4ego t'11 a[tso yee 1k1 an7daalwo'd00 &lt;Bilag1ana k'ehj7&gt;ß"
 
+# data.translations.footer.identifier_more_info
 msgid "Looking for U.S. government information and services? @link."
 msgstr "W1ashindoon da'7n77shj77sh hane' d00 bee 1k1 an7da'awo' han7t1h? @link"
 
+# data.translations.footer.identifier_more_info__1
 msgid "Visit USA.gov"
 msgstr "USA.gov d7n77['88["
 
+# data.translations.footer.identifier_aria
 msgid "Agency identifier"
 msgstr "Da'7n77sh bee 44h0zin7g77"
 
+# data.translations.footer.identifier_gsa_txt_aria
 msgid "Agency description"
 msgstr "Da'7n77sh baa hane'"
 
+# data.translations.footer.identifier_link_aria
 msgid "Important links"
 msgstr "Da'7l98go dahane'gi"
 
+# data.translations.footer.identifier_more_info_aria
 msgid "Government information and services"
 msgstr "W1ashindoonj7 dahane' d00 bee 1k1 an7da'awo'"
 
-#contact
+# data.translations.footer.eac_text
 msgid "in partnership with"
 msgstr "yi[ nidaalnish"
 
+# data.translations.footer.eac_logo_alt_text
 msgid "U.S. Election Assistance Commission logo which indicates a partnership with Vote.gov"
 msgstr "U.S. Election Assistance Commission bee bik'7'diidlid7g77 47 biniy4h7g77 Vote.gov yi[ ah77'1azhgo yaa halne'"
 
+# data.translations.footer.twitter_text
 msgid "Follow Vote.gov on Twitter"
 msgstr "Vote.gov bik44' nanin1 Twitter biyi'"
 
-#search
+# data.translations.footer.section_sign_up
+msgid "Email sign-up"
+msgstr "Email sign-up"
+
+# data.translations.footer.section_contact
+msgid "Contact"
+msgstr "Contact"
+
+# data.translations.footer.gsa_logo_alt
+msgid "GSA logo"
+msgstr "GSA logo"
+
+# data.translations.homepage.search_placeholder
 msgid "Search Vote.gov"
-msgstr "vote.gov hanit1h"
+msgstr "vote.gov"
```

Comment:
this should be 
`msgstr "vote.gov hanit1h"`

Reply:
added missing translation.